src/XMakeBuildEngine/Evaluation/Expander.cs

```diff
@@ -3341,12 +3341,12 @@ private static Type GetTypeFromAssembly(string typeName, string candidateAssembl
 
                 // Try to load the assembly with the computed name
 #if FEATURE_GAC
-#pragma warning disable 618
+#pragma warning disable 618, 612
```

Comment:
Okay, but we'll need this here till we start always building against the the next mono version.